advanced-integration/README.md

```diff
@@ -2,6 +2,14 @@
 
 ## Instructions
 
+### Using GitHub Codespaces
+
+1. Rename `.env.example` to `.env` and update `CLIENT_ID` and `APP_SECRET` - It will automatically restart the server.
```

Comment:
Can we use declarative secrets in the devcontainer instead of the manual step to rename the `.env` file and replace the secrets in the file? With declarative secrets, users will be prompted to add the secret at the time of codespace creation which would avoid the need to configure these env vars manually.

Reply:
Updated, thanks!